doc/sphinx_source/mainDocs/tcl-commands.rst

```diff
@@ -1616,7 +1616,7 @@ connect <host> <[+]port>
 listen <port> <type> [options] [flag]
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-  Description: opens a listening port to accept incoming telnets; type must be one of "bots", "all", "users", "script", or "off". Prefixing the port with a plus sign will make eggdrop accept SSL connections on it.
+  Description: opens a listening port to accept incoming telnets; type must be one of "bots", "all", "users", "script", or "off". Prefixing the port with a plus sign will make eggdrop accept SSL connections on it, prefixing with a minus sign will disallow secure connections.
```

Comment:
Is there ever a case where we want users to be able to disallow using secure connection? I appreciate the thoroughness of the change, but is modifying the listen command with the same syntax perhaps too far?

Reply:
uhm, yes, imagine localhost connections on your own dedicated box where secure localhost connections are just a waste of cpu cycles...

Reply:
Hi key2peace, thanks for the comment. This was not directed at the general concept of disallowing TLS, rather the specific case of with listen. Separate from the listen, botllinks can disable TLS using a - prefix in the .botaddr entry, which was the original intent of this PR, and would still allow you to disable a secure link on a per-bot basis.

Reply:
When you enfore no TLS on the listening port of a hub, there's no starttls initiated by the hubbot and thus a little bit less noise (and cpu cycles both on the hub and the slaves).